src/services/menu_creator_manager.py

```diff
@@ -563,7 +565,7 @@ def create_player_menu(
                         ],
                     )
                     chest_option = True
-                if "lock_picking" in player.skills and not pick_lock_option:
+                if "lock_picking" in player.skills and not entity.opened and not pick_lock_option:
```

Comment:
Could you remove this change that is for the other bug fix, in order to keep PRs more atomic?